doc/changes/changes-2.1.0.md

```diff
@@ -0,0 +1,21 @@
+# Exasol Test Containers 2.1.0, released 2020-07-03
+
+In this release we added support for downloading a file from BucketFS (e.g. in case you need a log from there).
+
+We also fixed a bug in the code that waits for an upload to complete that occurred when replacing an existing file. The call now properly blocks until the file is really replaced.
```

Comment:
I have checked some example on the net and realized that they go even further. They write from the perspective of the present. I think I am going to adopt that style, since from a user's perspective this is closest to what is of interest.

Reply:
Only in case of bugs I will still mention the past, so that users understand was went wrong before the fix.